selfdrive/modeld/models/commonmodel.pxd

```diff
@@ -0,0 +1,24 @@
+# distutils: language = c++
+#cython: language_level=3
```

Comment:
don't think it's necessary to specify `language_level` anymore, let's remove them